src/main/java/com/amazon/aocagent/commands/CommonOption.java

```diff
@@ -36,6 +37,12 @@
       defaultValue = ".aoc-stack.yml")
   private String stackFilePath;
 
+  @CommandLine.Option(
+      names = {"-e", "--extra-context"},
+      description = "eg, -e launchType=ec2 -e deployMode=sidecar",
```

Comment:
i feel we need to align the parameter value format, like all-uppercase, all-lowercase, or Camelcase？

Reply:
updated to all uppercase since these are treated as constants in ENUM